src/config.ts

```diff
@@ -202,6 +166,49 @@ export interface VLOnlyConfig<ES extends ExprRef | SignalRef> {
    * Allow the `formatType` property for text marks and guides to accept a custom formatter function [registered as a Vega expression](https://vega.github.io/vega-lite/usage/compile.html#format-type).
```

Comment:
Should customFormatType stay in `VLOnlyConfig`? (Users shouldn't have to turn it on twice?)  But maybe keeping it here simplify the doc.